src/commands/apply/index.ts

```diff
@@ -61,34 +73,22 @@ export default class Apply extends Command {
       environmentId: flags.environment,
       inputPath: args.input,
       changeset: createChangeset(flags.source, flags.target),
+      processedEntities: {
+        entries: { added: [], deleted: [], updated: [] },
+      },
     }
 
-    console.log(chalk.underline.bold('\nStart applying changeset'))
+    console.log(OutputFormatter.headline(`\nStart applying changeset to ${chalk.yellow(flags.environment)} 📥`))
 
-    const startTime = performance.now()
-    const result = await applyChangesetTask(context).run()
-    const endTime = performance.now()
+    // const startTime = performance.now()
+    await applyChangesetTask(context).run()
+    // const endTime = performance.now()
 
-    const duration = ((endTime - startTime) / 1000).toFixed(1)
-    const usedMemory = process.memoryUsage().heapUsed / 1024 / 1024
+    // const duration = ((endTime - startTime) / 1000).toFixed(1)
```

Comment:
Do we plan to bring this duration information back?

Reply:
Yes for analytics, that's why I kept it in for now